Release Notes for August 11 2015

catong opened the pull request:
Release Notes for August 11, 2015
Product/Engineering reviewers have been tagged inline. 
@lamagnifica is the doc reviewer.
FYI @mhoeber

catong commented on en_us/release_notes/source/2015/analytics/analytics_0811_2015.rst:
```diff
@@ -0,0 +1,3 @@
+
+The new student historical course enrollment table captures the history of a
+student’s orders and enrollment states. (XCOM-442)
```
@clintonb Is this description OK?

clintonb replied:
Just enrollment states. Order history is not tracked by this change.

-- 
Clinton Blackburn

clintonb replied:
Only enrollment history. The change does not track order history.

catong commented on en_us/release_notes/source/2015/lms/lms_0811_2015.rst:
```diff
@@ -0,0 +1,13 @@
+The bulk email template has been modified so that the link for “course email
+settings” now directs users to the dashboard rather than to the Account
+Settings page. (TNL-2677)
+
```
@explorerleslie please review the bug fix descriptions in the LMS and Studio files. Thanks!

explorerleslie replied:
The button text is "Modify course email settings." FWIW. Otherwise :+1: 
![image](https://cloud.githubusercontent.com/assets/5643426/9184079/0cae78b6-3f81-11e5-89ba-74218a04761a.png)

catong commented on en_us/release_notes/source/2015/studio/studio_0811_2015.rst:
```diff
@@ -0,0 +1,9 @@
+completed successfully. (TNL-2669)
+
+In problems with hints, MathJax did not display correctly in hints. This
+problem has been resolved. (TNL-2857, TNL-298)
```
@explorerleslie

explorerleslie replied:
:+1:

catong commented:
@explorerleslie Thanks, fixed the button name

mhoeber commented on en_us/release_notes/source/2015/studio/studio_0811_2015.rst:
```diff
@@ -0,0 +1,9 @@
+or back slashes. This problem has been resolved. If a course import or export
+is attempted and forward or back slashes are detected, these characters are
+replaced with an underscore character so that the import or export can be
+completed successfully. (TNL-2669)
```
dumb question - does this modify references to the asset?  would something break on later import?  should we check this at asset upload?

catong replied:
@adampalay Can you please confirm whether these changes in TNL-2669 should be further clarified, per @mhoeber 's questions?

adampalay replied:
That's a really good question. I think (but am not 100% sure) if you had a course with one of these bad references in it, and exported and imported, then that reference would break. @waheedahmed , am I understanding this correctly?

@catong and @mhoeber , the only way a course can end up in this state is if they export the course, and change the asset names in XML. If you try to upload an asset with a slash in its name in studio, that slash will be converted to an underscore. So it's very rare that a course will run into this use case.

waheedahmed replied:
@adampalay Yes I think so.

catong commented on en_us/release_notes/source/2015/studio/studio_0811_2015.rst:
```diff
@@ -0,0 +1,24 @@
+or review any support tickets you have created, go to https://partners.edx.org
+and select *Course Staff Support* at the top of the page. This option is
+available on every page in the Partner Portal.
+
```
@mhoeber Portal support item - can you please review?

mhoeber replied:
Course Staff Support -- bold, not ital.  otherwise :+1:

catong commented:
@clintonb @adampalay @mhoeber I have addressed your comments. @explorerleslie FYI in case you want to take a look at the revised description for TNL-2669.

lamagnifica commented on en_us/release_notes/source/2015/analytics/analytics_0811_2015.rst:
```diff
@@ -0,0 +1,3 @@
+
+The new student historical course enrollment table captures students'
+enrollment histories. (XCOM-442)
```
where do you find this table? is there a report/ CSV? is it for course teams and accessible from the Instructor Dashboard?
or, is there a new SQL table included in data packages for researchers to use? (if so, what's the name of the table?)
I'd include an xref to more info at a minimum.

clintonb replied:
At present this is only a SQL table. I am unaware of plans to include this in data packages.

lamagnifica replied:
I'm not sure what the purpose of announcing its existence is, @clintonb. Is there any way that anyone outside of edX could access this data?

clintonb replied:
Our data? No. An open source user could use a MySQL client to access their own data if they wanted. I am fine with removing the announcement if you don't think it's useful. There Analytics Team's usage of the data (or potential addition to the data packages) might be a more appropriate time.

lamagnifica replied:
@shnayder @jab5569 IYO is this a good time to announce the existence of a student historical course enrollment table (SQL, I don't have the name)?

shnayder replied:
I don't think this needs to be in release notes.

lamagnifica commented on en_us/release_notes/source/2015/lms/lms_0811_2015.rst:
```diff
@@ -0,0 +1,13 @@
+
+In problems with hints, MathJax did not display correctly in hints. This
```
I'd strike one of the references to hints, or rephrase as "In problems with hints, MathJax formatting did not display correctly in the hint text.

lamagnifica commented on en_us/release_notes/source/2015/lms/lms_0811_2015.rst:
```diff
@@ -0,0 +1,13 @@
+problem has been resolved. (TNL-2857, TNL-298)
+
+The bulk email template has been modified so that the link for the “Modify
+course email settings” button now directs users to the dashboard rather than
```
change curly quotes to straight quotes

lamagnifica commented on en_us/release_notes/source/2015/studio/studio_0811_2015.rst:
```diff
@@ -0,0 +1,25 @@
+Studio Support Integrated into Partner Portal
+=============================================
+
+For edX Partners, edX has incorporated technical support for course teams into
```
I'd lc "partners"

lamagnifica commented on en_us/release_notes/source/2015/studio/studio_0811_2015.rst:
```diff
@@ -0,0 +1,25 @@
+
+For edX Partners, edX has incorporated technical support for course teams into
+the edX Partner Portal. To access technical support, submit a support ticket,
+or review any support tickets you have created, go to https://partners.edx.org
```
https://partners.edx.org --> the edX Partner Portal (linked to https://partners.edx.org)

lamagnifica commented on en_us/release_notes/source/2015/studio/studio_0811_2015.rst:
```diff
@@ -0,0 +1,25 @@
+the edX Partner Portal. To access technical support, submit a support ticket,
+or review any support tickets you have created, go to https://partners.edx.org
+and select **Course Staff Support** at the top of the page. This option is
+available on every page in the Partner Portal.
```
^edX Partner Portal

lamagnifica replied:
Are the links to support removed from the footer/sock in Studio? is the partner portal the only way to access support now, or are the Studio links updated to take you to the portal? (if so, are the links only changed for edge & edx.org? if not, how does that affect open source sites, will they have to be sure to change those links?)

mhoeber replied:
I don't believe this is complete yet, but goal is to update Studio so that on edX the footer points to portal for course team support, and that the link is configurable for open source. But no changes to studio in this release.

lamagnifica replied:
perhaps then a reassurance that support tickets opened through the Studio links will continue to receive attention (if that is true)

lamagnifica commented on en_us/release_notes/source/2015/studio/studio_0811_2015.rst:
```diff
@@ -0,0 +1,25 @@
+when they are uploaded to Studio. (TNL-2669)
+
+In problems with hints, MathJax did not display correctly in hints. This
+problem has been resolved. (TNL-2857, TNL-298)
```
same as above

lamagnifica commented:
some suggestions to expand descriptions a bit

catong commented:
@lamagnifica incorporated your review comments. @jab5569 or @shnayder can you please confirm whether XCOM-442 should be included or removed? Thanks.

lamagnifica commented on en_us/release_notes/source/2015/lms/lms_0811_2015.rst:
```diff
@@ -0,0 +1,13 @@
+problem has been resolved. (TNL-2857, TNL-298)
+
+The bulk email template has been modified so that the link for the "Modify
+course email settings" button now directs email recipients to the dashboard
```
should this be bolded as a UI element name rather than quoted?

catong commented:
@shnayder Thanks. XCOM-442 removed.

lamagnifica commented:
:+1:

catong commented:
Merging https://github.com/edx/edx-documentation/pull/474
